packages/office-ui-fabric-react/src/components/TextField/TextField.test.tsx

```diff
@@ -82,513 +94,577 @@ describe('TextField', () => {
       }
     };
     const component = renderer.create(
-      <TextField
-        label="Label"
-        errorMessage={'test message'}
-        underlined={true}
-        prefix={'test prefix'}
-        suffix={'test suffix'}
-        styles={styles}
-      />
+      <TextField label="Label" errorMessage="test message" underlined={true} prefix="test prefix" suffix="test suffix" styles={styles} />
     );
     const tree = component.toJSON();
     expect(tree).toMatchSnapshot();
   });
+}); // end snapshots
+
+describe('TextField rendering values from props', () => {
```

Comment:
A lot of the tests are reorganized, and anything that was verifying incorrect behavior is removed